jumpscale/packages/vdc_dashboard/chats/matic.py

```diff
@@ -0,0 +1,58 @@
+from jumpscale.sals.chatflows.chatflows import StopChatFlow, chatflow_step
+from jumpscale.packages.vdc_dashboard.sals.solutions_chatflow import SolutionsChatflowDeploy
+from jumpscale.packages.vdc_dashboard.sals.vdc_dashboard_sals import get_deployments
+
+
+class MaticDeploy(SolutionsChatflowDeploy):
+    SOLUTION_TYPE = "matic"
+    title = "Matic"
+    HELM_REPO_NAME = "marketplace"
+    steps = ["get_release_name", "create_subdomain", "set_config", "install_chart", "initializing", "success"]
+
+    CHART_LIMITS = {
```

Comment:
According to matic i think that's quite low
![image](https://user-images.githubusercontent.com/64129/110297074-c09d8680-7ffb-11eb-8b0f-6586468e204c.png)

even the backing disk is not enough

Reply:
No problem, let me check

Reply:
So we have the improved the limits https://github.com/threefoldtech/js-sdk/pull/2724/commits/7d39f0e1b93d73d05223afb1c07e07c8ddedfd8e

For Disk size, PR is here https://github.com/threefoldtech/vdc-solutions-charts/pull/107